packages/react/src/components-shared/Toolbar/ToolbarInputDropdownButton.tsx

```diff
@@ -26,22 +25,20 @@ export const ToolbarInputDropdownButton = (
   }, []);
 
   return (
-    <Tippy
-      onShow={(instance) => {
+    <Popover
+      withinPortal={false}
+      onOpen={() => {
         createDropdown();
-        props.onShow?.(instance);
       }}
-      onHidden={(instance) => {
+      onClose={() => {
         destroyDropdown();
-        props.onShow?.(instance);
       }}
-      content={renderDropdown ? props.children[1] : null}
-      trigger={props.visible === undefined ? "click" : undefined}
-      interactive={true}
-      maxWidth={500}
-      zIndex={9000}
+      zIndex={10000}
       {...props}>
-      {props.children[0]}
-    </Tippy>
+      <Popover.Target>{props.children[0]}</Popover.Target>
```

Comment:
not necessarily related to this PR, but the pattern children[0] and children[1] is not very nice imo